development/index.js

```diff
@@ -27,6 +27,18 @@ require (['bigscreenplayer/bigscreenplayer'], function(BigscreenPlayer){
       }
     }
 
+  function setUpCaptionsContainerCSS() {
+    var captionsContainer = document.getElementById('playerCaptionsContainer');
+    captionsContainer.style.position = 'absolute';
+    captionsContainer.style.top = '80%';
+    captionsContainer.style.right = '50%';
+  }
+  
+  function normaliseVolume () {
+    var videoElement = document.getElementsByTagName('video')[0];
+    videoElement.volume = 0.1;
```

Comment:
What is this change for? When we tested this in rDot we never saw a value less than 1. Also, why are we setting it to the lowest possible value? This would make us inconsistent with other players and normal linear tv. 'Playing normal TV with acceptable volume and no problem, go to iplayer and it is too low, so turn it up. Go back to normal TV and it is deafeningly loud!'

Reply:
Well it is the dev environment script. The test asset I was using was rather loud when trying to remote pair with headphones.

Reply:
Didn't notice the dev script :D